apps/finance/app/src/components/Transfers.js

```diff
@@ -179,253 +174,110 @@ class Transfers extends React.PureComponent {
-                  </Filters>
-                )}
-              </Spring>
-            </Header>
-            {filteredTransfers.length === 0 ? (
-              <NoTransfers>
-                <p>
-                  No transfers found.{' '}
-                  {filtersActive && (
-                    <a role="button" onClick={this.handleResetFilters}>
-                      Reset filters
-                    </a>
-                  )}
-                </p>
-              </NoTransfers>
-            ) : (
-              <div>
-                <FixedTable
-                  header={
-                    above('medium') && (
-                      <TableRow>
-                        <DateHeader title="Date" />
-                        <SourceRecipientHeader title="Source / Recipient" />
-                        <ReferenceHeader title="Reference" />
-                        <AmountHeader title="Amount" align="right" />
-                        <TableHeader />
-                      </TableRow>
-                    )
-                  }
-                >
-                  {filteredTransfers
-                    .sort(({ date: dateLeft }, { date: dateRight }) =>
-                      // Sort by date descending
-                      compareDesc(dateLeft, dateRight)
-                    )
-                    .slice(0, displayedTransfers)
-                    .map(transfer => (
-                      <TransferRow
-                        key={transfer.transactionHash}
-                        token={tokenDetails[toChecksumAddress(transfer.token)]}
-                        transaction={transfer}
-                        smallViewMode={below('medium')}
-                      />
-                    ))}
-                </FixedTable>
-                {displayedTransfers < filteredTransfers.length && (
-                  <Footer>
-                    <Button mode="secondary" onClick={this.showMoreTransfers}>
-                      Show Older Transfers
-                    </Button>
-                  </Footer>
-                )}
-              </div>
+      <section>
+        <Header>
+          <Title compactMode={compactMode}>
+            <span>Transfers </span>
+            <span>
+              {compactMode && (
+                <ToggleFiltersButton
+                  title="Toggle Filters"
+                  onClick={this.handleToggleFiltersClick}
+                />
+              )}
+            </span>
+          </Title>
+          <TransfersFilters
+            dateRangeFilter={selectedDateRange}
+            onDateRangeChange={this.handleDateRangeChange}
+            tokenFilter={selectedToken}
+            onTokenChange={this.handleTokenChange}
+            transferTypeFilter={selectedTransferType}
+            onTransferTypeChange={this.handleTransferTypeChange}
+            compactMode={compactMode}
+            opened={filtersOpened}
+            symbols={symbols}
+            transferTypes={TRANSFER_TYPES_STRING}
```

Comment:
Could we rename `symbols` to `tokenSymbols`, and move it together with the token filter and onchange handler?

Similarly, could we move `transferTypes` to be with the transfer type filter and onchange handler?